app/default/application_routes.py

```diff
@@ -80,62 +128,69 @@ def tasklist(application_id):
-            "completed_status": ApplicationStatus.COMPLETED.name,
-            "submitted_status": ApplicationStatus.SUBMITTED.name,
-            "number_of_forms": len(application.forms),
-            "number_of_completed_forms": len(
-                list(
-                    filter(
-                        lambda form: form["status"]
-                        == ApplicationStatus.COMPLETED.name,
-                        application.forms,
-                    )
-                )
-            ),
-        }
-
+    account = get_account(account_id=application.account_id)
+    if application.status == ApplicationStatus.SUBMITTED.name:
         with force_locale(application.language):
             return render_template(
-                "tasklist.html",
-                application=application,
-                sections=sections,
-                application_meta_data=application_meta_data,
-                form=form,
-                contact_us_email_address=round_data.contact_details["email_address"],
-                submission_deadline=round_data.deadline,
-                is_past_submission_deadline=current_datetime_after_given(round_data.deadline),
+                "application_submitted.html",
+                application_id=application.id,
+                application_reference=application.reference,
+                application_email=account.email,
             )
-    return redirect(url_for("account_routes.dashboard"))
+    fund = get_fund_data(application.fund_id, as_dict=True)
+    sections = application.get_sections()
+    form = FlaskForm()
+    application_meta_data = {
+        "application_id": application_id,
+        "fund_name": fund.name,
+        "round_name": round_data.title,
+        "not_started_status": ApplicationStatus.NOT_STARTED.name,
+        "in_progress_status": ApplicationStatus.IN_PROGRESS.name,
+        "completed_status": ApplicationStatus.COMPLETED.name,
+        "submitted_status": ApplicationStatus.SUBMITTED.name,
+        "number_of_forms": len(application.forms),
+        "number_of_completed_forms": len(
+            list(
+                filter(
+                    lambda form: form["status"]
+                    == ApplicationStatus.COMPLETED.name,
+                    application.forms,
+                )
+            )
+        ),
+    }
+
+    with force_locale(application.language):
+        return render_template(
+            "tasklist.html",
+            application=application,
+            sections=sections,
+            application_meta_data=application_meta_data,
+            form=form,
+            contact_us_email_address=round_data.contact_details[
+                "email_address"
+            ],
+            submission_deadline=round_data.deadline,
+            is_past_submission_deadline=current_datetime_after_given_iso_string(  # noqa:E501
+                round_data.deadline
+            ),
+        )
+    # return redirect(url_for("account_routes.dashboard"))
```

Comment:
Should this still be commented out?

Reply:
Thanks - yes it should, it's left from before I changed the round open check to being a decorator.